deepcell_label/url_loaders_test.py

```diff
@@ -143,7 +143,7 @@ def test_load_raw_png():
 def test_load_raw_zip():
     """Load raw array from zip of tiff files."""
     expected_raw = np.array([1, 2]).reshape((1, 1, 1, 2))
-    expected_labeled = np.array([0, 0]).reshape((1, 1, 1, 2))
+    expected_labeled = np.array([0]).reshape((1, 1, 1, 1))
```

Comment:
Could you briefly comment on why this is changed?

Reply:
I think the behavior of creating a project when no labels are present has changed from having the same same shape as the raw input vs. now only having a single feature. Previously, loading a raw image with two channels would generate a label array with two features, but now it's just one feature. Having the numbers of features change on with the number of channels felt like a weird dependency